scripts/f5-config.py

```diff
@@ -14,55 +14,155 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
-
 import argparse
 import json
 import os
 
 import netaddr
 
+
+PART = 'RPC'
+PREFIX_NAME = 'RPC'
+
 SNAT_POOL = (
-    'create ltm snatpool %(prefix_name)s_SNATPOOL { members replace-all-with {'
-    ' %(snat_pool_addresses)s%(route_domain)s } }'
+    '### CREATE SNATPOOL ###\n'
+    'create ltm snatpool /' + PART + '/' + PREFIX_NAME + '_SNATPOOL { members replace-all-with {'
+    ' %(snat_pool_addresses)s } }'
 )
 
-MONITORS = (
-    'create ltm monitor mysql %(prefix_name)s_MON_GALERA { count 0 database'
-    ' information_schema debug yes defaults-from mysql destination *:*'
-    ' interval 30 time-until-up 0 timeout 91 username haproxy }'
-)
+#Persistance Profile:
+PERSISTANCE = [
+    r'create ltm persistence source-addr /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_IP {'
+    r' app-service none defaults-from /Common/source_addr'
+    r' match-across-services enabled timeout 3600 }',
+    r'create ltm persistence cookie /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_COOKIE {'
+    r' app-service none cookie-name RPC-COOKIE defaults-from /Common/cookie }''\n'
+]
+
+MONITORS = [
+    r'create ltm monitor mysql /' + PART + '/' + PREFIX_NAME + '_MON_GALERA { count 1 database'
+    r' information_schema debug no defaults-from mysql destination *:*'
+    r' interval 3 recv big5_chinese_ci recv-column 2 recv-row 0 send "select'
+    r' * from CHARACTER_SETS;" time-until-up 0 timeout 10 username haproxy }',
+    r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_KEYSTONE_ADMIN { defaults-from'
+    r' http destination *:35357 recv "200 OK" send "HEAD /v3 HTTP/1.1\r\nHost:'
+    r' rpc\r\n\r\n" }',
+    r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_NOVA_API_METADATA {'
+    r' defaults-from http destination *:8775 recv "200 OK" send "HEAD /'
+    r' HTTP/1.1\r\nHost: rpc\r\n\r\n" }',
+    r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_HORIZON { defaults-from http'
+    r' destination *:80 recv "302 Found" send "HEAD / HTTP/1.1\r\nHost:'
+    r' rpc\r\n\r\n" }',
+    r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_NOVA_SPICE_CONSOLE {'
+    r' defaults-from http destination *:6082 recv "200 OK" send "HEAD /'
+    r' HTTP/1.1\r\nHost: rpc\r\n\r\n" }',
+    r'create ltm monitor https /' + PART + '/' + PREFIX_NAME + '_MON_HTTPS_HORIZON_SSL { defaults-from'
+    r' https destination *:443 recv "302 FOUND" send "HEAD / HTTP/1.1\r\nHost:'
+    r' rpc\r\n\r\n"}',
```

Comment:
Place a space after the " so it reads " }'. This will aid in parsing the output with the F5api.